packages/-ember-data/tests/integration/adapter/queries-test.js

```diff
@@ -107,15 +107,18 @@ module('integration/adapter/queries - Queries', function(hooks) {
   });
 
   testInDebug('The store asserts when query is made and the adapter responses with a single record.', function(assert) {
-    env = setupStore({ person: Person, adapter: DS.RESTAdapter });
+    env = setupStore({
+      person: Person,
+      adapter: DS.RESTAdapter,
+    });
     store = env.store;
     adapter = env.adapter;
 
     adapter.query = function(store, type, query, recordArray) {
       assert.equal(type, Person, 'the query method is called with the correct type');
 
       return resolve({
-        data: [{ id: 1, type: 'person', attributes: { name: 'Peter Wagenet' } }],
+        data: { id: 1, type: 'person', attributes: { name: 'Peter Wagenet' } },
```

Comment:
This change seems very suspect?

Reply:
The test is currently wrong and happens to be relying on the RESTSerializer returning null